### PR TITLE
Add description of lab meetings

### PR DESCRIPTION
**Pull request by KirstieJane**
This big document includes some expectations and processes for lab meetings.

I'd love to hear your thoughts!

### CODE REVIEW

File: Lab-meetings.md [hunk 1]
```diff
@@ -0,0 +1,347 @@
...
+You can see the raw markdown and the rendered page side by side by clicking the `both` button.
+You can read more about HackMD at [https://hackmd.io/s/features](https://hackmd.io/s/features).
+
+The template for each lab meeting is below, and you can read more about why we have this format in the [motivation](#motivation) section at the end of this page.
+
+When you receive the HackMD link, please add your name under **Attendees** or **Apologies**.
+This lets the chair know that you've received the email and acknowledges that week's meeting.
+It also means the chair knows who to wait for at the start of the meeting.
+
+In advance of the meeting, please also complete the sections on **What do you want the lab members to know about?** and **What would you like lab members' opinion(s) on?**
```
Comment (ajrcampbell) [inline Lab-meetings.md:36]: So a lab meeting is about sharing what we are _individually_ working on, as opposed to setting a group project or sharing progress on something we are _collectively_ working on?

File: Lab-meetings.md [hunk 2]
```diff
@@ -0,0 +1,347 @@
+# Lab Meetings
+
+As the lab grows, the importance of sharing technical skills, domain expertise and tips and tricks for working effectively grows with it.
+Lab meetings are a space for all members of the group to come together and learn from each other.
+
```
Comment (ajrcampbell) [inline Lab-meetings.md:5]: I think a section on the aims or outcomes of a lab meeting would be useful here

File: Lab-meetings.md [hunk 3]
```diff
@@ -0,0 +1,347 @@
...
+[Small world networks](https://en.wikipedia.org/wiki/Small-world_network) in particular are efficient and resilient.
+
+![](https://upload.wikimedia.org/wikipedia/commons/thumb/2/2e/Decentralization_diagram.svg/300px-Decentralization_diagram.svg.png)
+
+*Decentralization diagram uploaded to [Wikimedia Commons](https://commons.wikimedia.org/wiki/File:Decentralization_diagram.svg) by Adam Aladdin. Reused under a CC-BY license.*
+
+I would like this lab to be as decentralised as possible.
+I would like lab members to **find the answers to their questions quickly and receive help and support from all other lab members** in all dimensions.
+
+One of the core goals of all Whitaker Lab is to improve collaboration (in academia and in the world!)
```
Comment (ajrcampbell) [inline Lab-meetings.md:210]: I think these core goals should go at the top!

File: Lab-meetings.md (hunk 1 above)
Comment (KirstieJane) [reply Lab-meetings.md:36]: Its a space to build connections between members of the group.

From my point of view it's really beneficial for folks to share skills and knowledge to enhance each other's separate projects. As I'm involved in almost all of them I can see clear overlaps and where skills and outputs can be re-used across different projects.....lab meeting is a space to try to make those clearer too.

I think starting a new project is brittle - most members of the group are flat out on their current projects so adding a shared one is likely to fail when everyone gets busy, and is going to be hard to find overlap for everyone's interests. Individual lab meetings could be used to work through specific skill and knowledge shares though!

Does that make sense? Is there a good place to add this clarification to the guidance?

File: Lab-meetings.md (hunk 2 above)
Comment (KirstieJane) [reply Lab-meetings.md:5]: Now that you've heard me present in the meeting, can you give adding those sentences a bash @ajrcampbell?

File: Lab-meetings.md (hunk 3 above)
Comment (KirstieJane) [reply Lab-meetings.md:210]: Sounds good! If you can open a PR to the lab-meetings branch that would be amazing! Thank you 🚀:star2:

**Review by yochannah — APPROVED** [review]
this is a long read but some amazing guidelines - I love it!

**Review by sgibson91 — APPROVED** [review]

**Review by LouiseABowler — APPROVED** [review]
I really like the guidelines @KirstieJane! A long ready but everything is very clear ✨

**Review by sparkler0323 — APPROVED** [review]
🎇

**Comment by KirstieJane** [comment]
@ajrcampbell - I'm going to merge this because I really want these guidelines in the main repo. But please fix forwards when you have time! Moving the content around to make it clearer would be great!

**Pull request merged**